src/app/constructor/dashboard/dashboard-constructor.component.ts

```diff
@@ -9,14 +10,35 @@ import { DashboardViewInfoDto } from '../../home/dashboard-view/dashboard-view.d
 })
 export class DashboardConstructorComponent {
     @Input() canBeActive: boolean;
-    @Input() default: string;
-    @Input() dashboardSubViewData: DashboardViewInfoDto;
+
+    @Output() defaultSubviewChange: EventEmitter<string> = new EventEmitter<string>();
+    @Input()
+    set defaultSubview(value) {
+        this.defaultSubviewValue = value;
+        this.defaultSubviewChange.emit(value);
```

Comment:
So much eventEmitters. Can you some data update by link without using of eventemitters?